cmd/init-repo/main.go

```diff
@@ -84,18 +89,9 @@ func main() {
 		glog.Fatalf("Target organization cannot be empty")
 	}
 
-	// If RULE_FILE_PATH is detected, check if the source repository include rules files.
-	if len(os.Getenv("RULE_FILE_PATH")) > 0 {
-		cfg.RulesFile = filepath.Join(BaseRepoPath, cfg.SourceRepo, os.Getenv("RULE_FILE_PATH"))
-	}
-
 	if len(cfg.RulesFile) == 0 {
 		glog.Fatalf("No rules file provided")
 	}
-	rules, err := config.LoadRules(cfg.RulesFile)
```

Comment:
@sttts I broke this into 3 separate functions each having its own logic to handle the sources...

With this PR we will handle reading of the rule file using:

```
--rules-file=/etc/foo/rules.yaml              # Absolute path
--rules-file=configs/rules.yaml               # Relative to source repository root (rules-file-branches affecting the branch)
--rules-file=https://foo.com/rules.yaml  # URL

```